appveyor.yml

```diff
@@ -123,6 +127,32 @@ branches:
   only:
     - auto
 
+after_test:
```

Comment:
Yeah, I have seen that too but you can `Push-AppveyorArtifact` the artifacts in the [before_deploy](https://github.com/japaric/trust/blob/master/ci/before_deploy.ps1#L18) phase. You still have to declare them in [appveyor.yml](https://github.com/japaric/trust/blob/master/appveyor.yml#L58) though but a glob works.